addons/variable-manager/style.css

```diff
@@ -0,0 +1,82 @@
+.sa-var-manager {
+  display: block;
+  padding: 18px;
+  /* weird hack to fix scrolling??? */
+  height: 50px;
+  overflow-y: auto;
+}
+
+.sa-var-manager-searchbox {
+  border: 1px solid #dddddd;
+  background: none;
+  background-image: url("search.svg");
+  padding: 8px;
+  padding-right: 5%; /* for the text to not overlap the image */
+  border-radius: 4px;
+  background-repeat: no-repeat;
+  background-size: 18px 18px;
+  background-position: 95% center;
+}
+
+.sa-var-manager.freeze > * > * > * > * > * {
```

Comment:
this is better than adding the same class to every element imo. it's ugly but atleast it works.

Reply:
@jeffalo [Beautiful is better than ugly.](https://www.python.org/dev/peps/pep-0020/#:~:text=Beautiful%20is%20better%20than%20ugly.) I don't see what's wrong with adding a class to every element.

Reply:
alright then. i think the main issue was figuring out how to get everything except the focused input to be greyed out. i don't know if i can do it in pure CSS so maybe I'll need to use js for adding the classes and handling all that. ill have time tomorrow to try

Reply:
> i think the main issue was figuring out how to get everything except the focused input to be greyed out. i don't know if i can do it in pure CSS

Something like this might work (but I haven't tested): `.sa-var-manager table:focus-within input:not(:focus) {color: gray;}`